src/app/containers/Include/amp/VjAmp.jsx

```diff
@@ -7,6 +7,39 @@ const IncludeGrid = styled(GridItemConstrainedMedium)`
   display: grid;
 `;
 
+const PocketTapArea = styled.div`
+  &&& {
+    display: block;
+    height: 70px;
+    background: linear-gradient(
+      0deg,
+      #fff 0%,
+      rgba(255, 255, 255, 0.816947) 66%,
+      rgba(255, 255, 255, 0.0718488) 100%
+    );
+    &:after {
+      display: inline-block;
+      content: '';
+      width: 100vw;
+      border-bottom: 2px solid #1380a1;
+      transform: translateY(2.4rem);
+    }
+  }
+`;
+
+const PocketButton = styled.button`
+  background: #1380a1;
```

Comment:
Once again re porting these styles from somewhere else, I think the colours should generally come from our colours in psammead, these have been agreed with UX: https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/colours.js